frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/mobile_card_detail_screen.dart

```diff
@@ -24,12 +25,16 @@ class MobileCardDetailScreen extends StatefulWidget {
   const MobileCardDetailScreen({
     super.key,
     required this.rowController,
+    required this.fieldController,
   });
 
   static const routeName = '/MobileCardDetailScreen';
   static const argRowController = 'rowController';
   static const argCellBuilder = 'cellBuilder';
+  static const argFieldController = 'fieldController';
```

Comment:
is it the required parameter for every card detail page?

Reply:
Yes it's needed for the `FieldEditorBloc`